examples/skeleton/skeleton.card.tsx

```diff
@@ -0,0 +1,54 @@
+import type { CodeData } from '~/components/code-demo';
+import { SkeletonCard } from '~/src';
+
+const code = `
+'use client';
+
+import { Skeleton } from 'flowbite-react';
+
+function Component() {
+  return (
+    <div>
+      <Skeleton.Card />
+    </div>
+  )
+}
+`;
+
+const codeRSC = `
+import { SkeletonVideo } from 'flowbite-react';
```

Comment:
Typo Fixed!.

@SutuSebastian --- for Video & Image Examples, I am thinking about how we can use the root skeleton? because in Image & Video Skeleton, I have the placeholder of SVG Icon for it. 

Please guide me.